src/lib/spsp.js

```diff
@@ -108,17 +115,19 @@ const quote = function * (plugin, {
   destinationAmount,
   connectors,
   id,
-  timeout
+  timeout,
+  spspResponse
 }) {
   assert(plugin, 'missing plugin')
-  assert(receiver, 'receiver')
+  assert(receiver || spspResponse,
```

Comment:
Should there be an assertion that checks that `spspResponse` is a valid SPSP Response?